.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+defaults: &defaults
+  docker:
+    - image: circleci/node:8
+  working_directory: ~/nodejs
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-yarn-cache-{{ checksum "yarn.lock" }}
```

Comment:
We had "issues" with the cache before. By just appending a version we can easily "bust" the cache.